packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -372,6 +372,46 @@ export default class ChildProcessWorker
         this._child.send(this._request);
       }
     } else {
```

Comment:
Note that this `else` branch has gone through several changes in the past.

- This PR added a check for `signal === "SIGABRT"`: https://github.com/facebook/jest/pull/13054 
- But was removed in a followup PR https://github.com/facebook/jest/pull/13056